sdk/src/main/scala/com/horizen/account/state/Account.scala

```diff
@@ -1,6 +0,0 @@
-package com.horizen.account.state
-
-case class Account(nonce: Long,
```

Comment:
The UT WithdrawalMsgProcessorTest.scala still uses this class, we can remove the testInit() there, we have the analogous UT which targets stateDb

Reply:
Done